sklearn/neighbors/binary_tree.pxi

```diff
@@ -293,11 +293,11 @@ Examples
 Query for k-nearest neighbors
 
     >>> import numpy as np
-
+    >>> from sklearn.neighbors import KDTree
```

Comment:
Hi maniteja123,
thanks for the comment. I'm not sure about the import statement and removed it for now, because it has nothing to do with the issue I attempted to fix about the deprecation warning. If there's something to do about one or more import statement, another Issue should be opened.